<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sekura::DataType</class>
 <widget class="QWidget" name="Sekura::DataType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Index</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DateTime</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Boolean</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Integer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Float</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>String</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>8192</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>MAX</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="5"/>
 <resources/>
 <connections/>
</ui>
